### DISCUSSION TITLE
Layout issue: repo description and star section are overlapping

### DISCUSSION THREAD

**Discussion by haleygelato** [community/community#120614] [Mobile]
### Select Topic Area

Bug

### Body

The repository description and star section are overlapping. This also occurs when you make any changes to the personal info section, but that overlapping would go away if you pull to refresh the page. However, the repo description here doesn't go away even after refreshing.

![DF0BC49E-E661-4D6C-8906-AA8AB9918965](https://github.com/community/community/assets/112786962/833f9e47-4bb1-4dda-bde2-e40def3d259b)

**Comment by haleygelato** [comment]
Okay, update: It actually got fixed after I made some changes to my bio section and refreshed...

**Comment by azooz71** [comment]
ا
